src/handlers/rendered_link.rs

```diff
@@ -52,7 +52,15 @@ async fn add_rendered_link(
                         .iter()
                         .any(|tf| f.filename.starts_with(tf))
             })
-            .and_then(|file| {
+            .collect();
+
+        // Sort the relavant files by the total number of lines changed, as to
+        // improve our guess for the relevant file to show the link to.
+        relevant_files.sort_by_key(|f| f.additions + f.deletions + f.changes);
```

Comment:
You can just use `max_by` instead of sorting the whole list and then taking the last element, but it probably doesn't matter that much here.

Reply:
Didn't know about `Iterator::max_by`, switch to it.